src/screens/scanning/ManualVoucherScreen.tsx

```diff
@@ -74,12 +75,27 @@ function ManualVoucherScreen() {
           </FieldContainer>
           <FieldContainer>
             <InputTitleText>Amount</InputTitleText>
-            <InputField
-              onChange={onChangeVoucherAmount}
+            <CurrencyInput
```

Comment:
Would we want to show the dollar sign prefix?

Reply:
yeah, i think this would be a good idea